### PR TITLE
MyInterests test

### PR DESCRIPTION
**Pull request by AleksandraPashchenko**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <scope>test</scope>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.141.0</version>
+        </dependency>
+
```
Comment (vladlukjanenko) [inline pom.xml]: Please, remove empty line

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    </dependencies>
+
```
Comment (vladlukjanenko) [inline pom.xml]: Also, empty line

Comment (AleksandraPashchenko) [reply pom.xml]: fixed

File: pom.xml (hunk 1 above)
Comment (AleksandraPashchenko) [reply pom.xml]: fixed

**Review by vladlukjanenko — APPROVED** [review]

**Pull request merged**